bin/run_robot

```diff
@@ -3,7 +3,7 @@
 #
```

Comment:
i basically modified that scriptie slightly since I recall having a problem where run_robot failed to run in some cases where the step was not queued -- perhaps that was fixed ?